appveyor.yml

```diff
@@ -36,9 +36,10 @@ install:
 build: off
 
 test_script:
+  - npm -g install npm@4
```

Comment:
Should we move this to the `install` section, where we are doing a `npm install -g npm@5.3 yarn` (probably replacing the `npm@5.3` part)?

Reply:
Great spot! I missed that completely, now fixed, thanks :+1: